internal/provider/resource_gitlab_instance_variable.go

```diff
@@ -24,40 +24,7 @@ var _ = registerResource("gitlab_instance_variable", func() *schema.Resource {
 			StateContext: schema.ImportStatePassthroughContext,
```

Comment:
I know it wasn't modified in this PR but I'd suggest changing the description of the resource to match the GitLab API docs like the other suggestions I've made:

```
Description: `The ` + "`" + `gitlab_instance_variable` + "`" + ` resource allows to manage the lifecycle of an instance-level CI/CD variable.
```

Reply:
done :)